whylog/tests/tests_rulesbase/__init__.py

```diff
@@ -0,0 +1,20 @@
+from unittest import TestCase
+from whylog.rulesbase import YamlConfigAccess
+
+
+class TestYamlConfigAccess(TestCase):
+
+    def test_should_return_parsers_with_regex(self):
+        #given
+        parsers_path = "whylog/tests/tests_rulesbase/test_files/simple_parsers.yaml" 
+        rules_path = "whylog/tests/tests_rulesbase/test_files/simple_rules.yaml"
+        config = YamlConfigAccess(parsers_path, rules_path)
+        #when
+        parsers = config.get_parsers()
+        #then
+        print parsers
```

Comment:
1. It shouldn't work on python 3. In python 2 print is keyword but in python 3 print is function. 
more about it here https://docs.python.org/3.0/whatsnew/3.0.html
'''
print(parsers)
'''
Sholud works well in both python 2 and 3.
2. Do we really need prints in the tests?